### ISSUE TITLE
1.12.2: AntiHeldItemChangeMod does not work

### ISSUE THREAD

**Issue by CRTCTY** [fr1kin/ForgeHax#431]
I think it uses the wrong packet

**Comment by 0-x-2-2** [comment]
Yeah, this was fixed years ago but apparently in a merge or rebase or something the change was lost. Change the packet to `SPacketHeldItemChange` and the code should work.

https://github.com/fr1kin/ForgeHax/commit/8e0e3d3e92f5932b3dc6913baf99c3f50f977852
